packages/conf-g++/conf-g++.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: "Francois Berenger"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git+https://github.com/ocaml/opam-repository.git"
```

Comment:
There's no need for `dev-repo` for conf packages.

Reply:
but opam lint gives me a warning if I remove it

Reply:
Really? What version of opam are you running. `opam lint` does not give a warning for no `dev-repo` if there's no `url`.

Reply:
I’m not an opam-repository maintainer so, no, I couldn’t edit your PR...

Reply:
Incidentally, the change to `opam lint` was made for the release candidate 8 months ago - you should upgrade!